src/cluster_legacy.c

```diff
@@ -7327,6 +7406,24 @@ int clusterCommandSpecial(client *c) {
     } else if (!strcasecmp(c->argv[1]->ptr, "links") && c->argc == 2) {
         /* CLUSTER LINKS */
         addReplyClusterLinksDescription(c);
+    } else if (!strcasecmp(c->argv[1]->ptr, "import") && c->argc > 3 && c->argc % 2 == 1) {
+        /* CLUSTER IMPORT SLOTSRANGE <start slot> <end slot> [<start slot> <end slot> ...] */
+        clusterCommandImport(c, /*one_shot=*/1);
+    } else if (!strcasecmp(c->argv[1]->ptr, "import-prepare") && c->argc > 3 && c->argc % 2 == 1) {
+        /* CLUSTER IMPORT-PREPARE SLOTSRANGE <start slot> <end slot> [<start slot> <end slot> ...] */
+        clusterCommandImport(c, /*one_shot=*/0);
+    } else if (!strcasecmp(c->argv[1]->ptr, "import-commit") && c->argc == 4) {
+        /* CLUSTER IMPORT-COMMIT LINK <link-name> */
+        clusterCommandImportCommit(c);
+    } else if (!strcasecmp(c->argv[1]->ptr, "migrations") && c->argc == 2) {
+        /* CLUSTER MIGRATIONS */
+        clusterCommandMigrations(c);
+    } else if (!strcasecmp(c->argv[1]->ptr, "import-cancel") && c->argc > 2) {
+        /* CLUSTER IMPORT-CANCEL (LINK <link-name>|ALL) */
+        clusterCommandImportCancel(c);
+    } else if (!strcasecmp(c->argv[1]->ptr, "syncslots") && c->argc > 2) {
```

Comment:
It is an internal command, I think you reference the command RESTORE-ASKING to create a separated function.